case-lib/hijack.sh

```diff
@@ -77,6 +77,7 @@ function exit()
         ;;
         *)
             dlogi "Unknown test exit code: $exit_status"
+            dlogi "Test Result: INCOMPLETE!"
```

Comment:
If this printout is not used as key words, it's better not to add. The original log is fine.

Reply:
current no code go to this logic, so it will be a hide bug when we catch `Test Result` as keyword